### PR TITLE
Feature/image quality enhancements

### PR DESCRIPTION
**Pull request by ethan605**
# Problem

# Solution

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [ ] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/ReduxAppWrapper/store/reducers/captures.ts
Comment (stefaniacardenas) [inline src/components/ReduxAppWrapper/store/reducers/captures.ts:9]: How many takes are we taking? By takes we mean "frames"? For how long do we store them in memory and will we delete them after upload?
I'm wondering if this might be taxing on low end devices memory...

File: src/locales/es_ES/es_ES.json
Comment (stefaniacardenas) [inline src/locales/es_ES/es_ES.json:149]: Is this feature going to be in English only? Is this documented in the README?

File: src/components/ReduxAppWrapper/store/reducers/captures.ts
Comment (stefaniacardenas) [inline src/components/ReduxAppWrapper/store/reducers/captures.ts:64]: Might be worth renaming the word `take` and `takes` to `frame` and `frames` so we know we are referring to the frames in multiframe capture features

File: src/components/Router/HistoryRouter.tsx
Comment (ethan605) [reply src/components/Router/HistoryRouter.tsx]: It was removed in the latest commit

File: src/components/Capture/Document.tsx
Comment (ethan605) [reply src/components/Capture/Document.tsx:291]: So far in this initiative: yes. For a public release: I don't know yet. Actually there's already a ticket to integrate `sdk-configuration` service and then we can dynamically control this enablement from backend side/feature flags.

File: src/locales/es_ES/es_ES.json
Comment (ethan605) [reply src/locales/es_ES/es_ES.json:149]: This will be documented in `MIGRATION.md`, but currently, we have plenty of keys waiting for release in the next version after `6.9.0`, I'll wait after that to avoid unnecessary conflicts.

File: src/components/ReduxAppWrapper/store/reducers/captures.ts
Comment (ethan605) [reply src/components/ReduxAppWrapper/store/reducers/captures.ts:9]: Sorry I wasn't making it easy to understand in the first place:

- Firstly, we don't store any "frame" in terms of binary data, but instead, we want to keep track of "how many times the user upload a photo of documents during the current verification session". So there'll barely any memory overhead there, the only data to be store is string timestamps.
- Secondly, (to answer your other question about their names): a "take" means when a document was captured (not necessarily uploaded yet), while a "frame" is more about the data itself. Apparently "takes" are purely for analytics events, something we really want to know about when comparing across user sessions, such as how many times on average a user need to capture & upload a "good quality" document.

File: src/components/ReduxAppWrapper/store/reducers/captures.ts
Comment (ethan605) [reply src/components/ReduxAppWrapper/store/reducers/captures.ts:64]: Answered in the above reply :point_up:

File: src/components/DocumentMultiFrame/index.tsx
Comment (ethan605) [reply src/components/DocumentMultiFrame/index.tsx:92]: Sorry, updated comment documentation for this: https://github.com/onfido/onfido-sdk-ui/blob/3e87294438a3184a77ee7fb455c7c4fb6ca05875/src/components/Overlay/DocumentOverlay.tsx#L201-L203

File: src/components/ReduxAppWrapper/store/reducers/captures.ts
Comment (stefaniacardenas) [reply src/components/ReduxAppWrapper/store/reducers/captures.ts:9]: Thank you!

File: src/locales/es_ES/es_ES.json
Comment (stefaniacardenas) [reply src/locales/es_ES/es_ES.json:149]: Ok, I will leave this conversation unresolved so you can remember to add details to `MIGRATION.md` before merging

**Review by stefaniacardenas — APPROVED** [review]
Looks good to me! Just remember to update `MIGRATION.MD` with new strings, apart from that it looks good 🚀

File: src/components/utils/http.ts
Comment (kopijunkie) [inline src/components/utils/http.ts:36]: Should this be tracked when it happens? I'm assuming this error is being caught in case back end service does somehow return a non-JSON response instead of what we're expecting.

File: src/locales/es_ES/es_ES.json
Comment (kopijunkie) [reply src/locales/es_ES/es_ES.json:149]: Since `6.9.0` copy updates has/should be merged in already, should `MIGRATION.md` be updated in this PR also now?

**Review by kopijunkie — CHANGES_REQUESTED** [review]

File: src/components/Confirm/Confirm.js
Comment (ethan605) [reply src/components/Confirm/Confirm.js:6]: Yep, it's different: https://github.com/onfido/onfido-sdk-ui/blob/49fb61229d6a2c65c4886e35e40f90fd451ba736/src/components/utils/onfidoApi.ts#L123

Here's the liveness video endpoint: https://github.com/onfido/onfido-sdk-ui/blob/49fb61229d6a2c65c4886e35e40f90fd451ba736/src/components/utils/onfidoApi.ts#L222

**Comment by stefaniacardenas** [comment]
Hello, I'm back 👋 
What's happening with this PR? Can it be merged?

**Comment by ethan605** [comment]
:wave: @stefaniacardenas this is currently on-hold due to our shifted focus to iOS & Android platforms this quarter

**Comment by ethan605** [comment]
Close due to the lack of active development on this feature

**Pull request closed without merging**
